src/ngraph/frontend/onnx_import/op/constant.hpp

```diff
@@ -0,0 +1,112 @@
+/*******************************************************************************
```

Comment:
Bob says these /* headers cause problems, so can you // them?

Reply:
done, according to review by @rkimballn1 #1169